Source/ui_libretro/main_libretro.cpp

```diff
@@ -493,6 +493,19 @@ bool retro_load_game(const retro_game_info* info)
 {
 	CLog::GetInstance().Print(LOG_NAME, "%s\n", __FUNCTION__);
 
+#if defined(IOS)
```

Comment:
Oh, are you sure `IOS` is defined, I'm not sure this is set in any CMake file or in the iOS SDK.

Reply:
I actually am, https://git.libretro.com/libretro-infrastructure/ci-templates/-/blob/master/ios-cmake.yml?ref_type=heads#L14, but I'm happy to change the preprocessor check to something like `#if TARGET_OS_IPHONE` if you prefer.

Reply:
Thanks very much!